frontend/packages/console-app/src/components/guided-tours/GuidedTourComponent.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { cloneDeep } from 'lodash';
+import { Button } from '@patternfly/react-core';
+import { GuidedTourItem, GuidedTourTaskItem } from './utils/guided-tour-typings';
+import { GuidedTourStatus } from './utils/guided-tour-status';
+import GuidedTourTask from './GuidedTourTask';
+
+type GuidedTourComponentProps = {
+  guidedTour: GuidedTourItem;
+  tourState?: GuidedTourStatus;
+};
+
+enum SubmitButtonText {
+  START = 'Start',
+  NEXT = 'Next',
+  FINISH = 'Finish',
```

Comment:
I think we need a back button

Reply:
the back button is always unchanged, so did not use the enum for that